salt/modules/boto_secgroup.py

```diff
@@ -255,8 +273,7 @@ def convert_to_group_ids(groups, vpc_id=None, region=None, key=None, keyid=None,
         else:
             log.debug('calling boto_secgroup.get_group_id for'
                       ' group name {0}'.format(group))
-            group_id = get_group_id(group, vpc_id, region, key, keyid,
-                                    profile, vpc_name)
+            group_id = get_group_id(group, vpc_id, vpc_name, region, key, keyid, profile)
```

Comment:
Same note for all other locations where you're calling a function where you've moved around the args.